File: control-flow/Q87/main.go
package main

import "fmt"

func main() {
	var x float64
	fmt.Print("Enter x: ")
	fmt.Scan(&x)
	sign := 1
	pow := 1.0
	sum := 0.0
	sum1 := 0.0
	for i := 1; i <= 10; i++ {
		pow = pow * x
		sum1 = sum1 + float64(i) * pow
		sum = sum + float64(sign) * 1.0 / sum1
		sign = -sign
	}
	fmt.Println("S =", sum)
}

File: control-flow/Q87/main.rs
use std::io;

fn main() {
    let mut input = String::new();

    println!("Enter x:");
    io::stdin().read_line(&mut input).expect("Failed to read line");
    let x: f64 = input.trim().parse().expect("Not a valid number");

    let mut sign: f64 = 1.0;
    let mut pow: f64 = 1.0;
    let mut sum: f64 = 0.0;
    let mut sum1: f64 = 0.0;

    for i in 1..=10 {
        pow *= x;
        sum1 += (i as f64) * pow;
        sum += sign * (1.0 / sum1);
        sign = -sign;
    }

    println!("S = {}", sum);
}